CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(EXTENSION_HOMEPAGE "http://slicer.org/slicerWiki/index.php/Documentation/Nig
 set(EXTENSION_CATEGORY "Examples")
 set(EXTENSION_CONTRIBUTORS "Jean-Baptiste Vimort (Kitware), Jean-Christophe Fillion-Robin (Kitware)")
 set(EXTENSION_DESCRIPTION "This extension allow to use a an HTC vive to display data in Virtual Reallity")
-set(EXTENSION_ICONURL "http://www.example.com/Slicer/Extensions/VR.png")
+set(EXTENSION_ICONURL "https://rawgit.com/KitwareMedical/SlicerOpenVR/master/VR.png")
```

Comment:
I don't think rawgit.com is required, the Content-Type for the url like seems correct https://raw.githubusercontent.com/KitwareMedical/SlicerOpenVR/master/VR.png

Reply:
Changed! :smiley_cat: